source/upgrade.txt

```diff
@@ -72,6 +72,23 @@ Version 3.0 Potential Breaking Change
 - The LINQ2 provider has been removed from this version of the driver.
   You must use LINQ3 for all LINQ queries.
 
+- Previous versions of the {+driver-short+} supported two GUID representation modes.
+  In version 3.0, ``GuidRepresentationMode.V3`` is the only supported mode. This change
+  has the following effects on the driver:
+  
+  - The methods in the ``BsonBinaryReader``, ``BsonBinaryWriter``, ``JsonReader``, and
+    ``JsonWriter`` classes no longer check the ``GuidRepresentationMode``.
```

Comment:
```suggestion
    ``JsonWriter`` classes no longer check the ``GuidRepresentationMode`` setting (?).
```

Reply:
On further review, this is probably a little more detail than the users need; it's pretty unlikely they would be depending on this check in this method. Following the steps on the GUID page will get them where they need to be.